modules/openy_features/openy_location/modules/openy_loc_branch/openy_loc_branch.install

```diff
@@ -377,3 +377,15 @@ function openy_loc_branch_update_8017() {
     }
   }
 }
+
+/**
+ * Update feature configs.
+ */
+function openy_loc_branch_update_8018() {
+  $config_dir = drupal_get_path('module', 'openy_loc_branch') . '/config/install/';
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
```

Comment:
I guess updating the config partially might work here better. See the preview update hook as an example.